pkg/rules/custom_test.go

```diff
@@ -0,0 +1,37 @@
+package rules
+
+import (
+	"testing"
+
+	"github.com/prometheus/prometheus/model/labels"
+	prometheus_promql "github.com/prometheus/prometheus/promql"
+	"github.com/stretchr/testify/require"
+
+	promscale_promql "github.com/timescale/promscale/pkg/promql"
+)
+
+func TestYoloVector(t *testing.T) {
```

Comment:
Would like to see a couple of more tests with edge cases since this is a yolo func. Thinks like empty values etc.